### PR TITLE
Delete Tenant Keys and More

### PR DESCRIPTION
**Pull request by jlmcgehee21**
Why:

* https://github.com/nerves-hub/nerves_hub_web/issues/29

This change addresses the need by:

* Adding a proper `tenant_key` controller, and accessing it's proper CRUD
endpoints from the tenant view.
* Testing this functionality.
* Testing the tenant controller.

Other things addressed:
* https://github.com/nerves-hub/nerves_hub_web/issues/127
* https://github.com/nerves-hub/nerves_hub_web/issues/72 (done via CDN
because I don't know enough frontend to package all of the assets
appropriately)

### CODE REVIEW

**Comment by mobileoverlord** [comment]
This is great, couple questions.
Do we ignore `priv/static` now?
I believe that this requires that we call `phx.digest` when building our prod container too right?

File: apps/nerves_hub_www/priv/static/js/app.js [hunk 1]
```diff
@@ -1,3 +1,120 @@
-// for phoenix_html support, including form and button helpers
-// copy the following scripts into your javascript bundle:
-// * https://raw.githubusercontent.com/phoenixframework/phoenix_html/v2.10.0/priv/static/phoenix_html.js
\ No newline at end of file
+/******/ (function(modules) { // webpackBootstrap
```
[the remark this answers is not in the document]
Comment (jlmcgehee21) [reply apps/nerves_hub_www/priv/static/js/app.js]: Like I said, I'm near illiterate when it comes to frontend asset management, but I think we can leave it out if we run `npm install` in prod.

**Comment by jlmcgehee21** [comment]
Check this commit message: https://github.com/nerves-hub/nerves_hub_web/pull/155/commits/65de3b1482159a51dfbf46b555f4010ec816bb61

There's a bit of hackery going on because we're redirecting back to the tenant `:edit` page to keep the flow consistent.  I would really like to have a `:show` view for the tenant, and render all the keys there (similar to what is happening now), but have an `add key` button, which takes you do a form that's rendered by the `tenant_key_controller`.

**Comment by mobileoverlord** [comment]
I think this is good to go. Thanks @stwf and @jlmcgehee21 ❤️

**Pull request merged**